packages/test/src/workflows/cancel-timer-immediately.ts

```diff
@@ -1,18 +1,23 @@
-import { CancellationError, cancel, sleep } from '@temporalio/workflow';
+// @@@SNIPSTART nodejs-cancel-a-timer-from-workflow
+import { CancelledError, CancellationScope, sleep } from '@temporalio/workflow';
 import { Empty } from '../interfaces';
 
 async function main(): Promise<void> {
-  const timer = sleep(3);
-  cancel(timer);
+  // Timers and Activities are automatically cancelled when their containing scope is cancelled.
   try {
-    await timer;
+    await CancellationScope.cancellable(async () => {
+      const promise = sleep(1); // <-- Will be cancelled because it is attached to `scope`
```

Comment:
```suggestion
      const promise = sleep(1); // <-- Will be cancelled because it is attached to this closure's scope
```

Maybe a bit more clear